src/sidebar.jsx

```diff
@@ -72,6 +72,16 @@ const getDescriptionListItems = selected => {
                     label: _("Size"),
                     value: cockpit.format_bytes(selected.size),
                 },
+                {
+                    id: "description-list-category",
+                    label: _("Category"),
+                    value: selected.category.name
+                },
+                {
+                    id: "description-list-icon",
+                    label: _("Icon"),
+                    value: selected.category.icon
+                },
```

Comment:
This is in no design, we just want icons in the list view and overview. Can we keep things simple please :)

Reply:
Ya.  This is purely debugging output.  When I said this is "proof of concept" I really meant it :)